eth/transactionManager_test.go

```diff
@@ -35,6 +35,9 @@ func (stm *stubTransactionSenderReader) TransactionByHash(ctx context.Context, t
 }
 
 func (stm *stubTransactionSenderReader) TransactionReceipt(ctx context.Context, txHash common.Hash) (*types.Receipt, error) {
+	if stm.receipt == nil {
+		return nil, ethereum.NotFound
+	}
```

Comment:
Could we have this as a separate commit / mentioned in the PR name?

Reply:
yeah, makes sense. If unrelated, always better to put in a separate commit (or even PR).